Source/WebCore/platform/graphics/coretext/FontPlatformDataCoreText.cpp

```diff
@@ -70,18 +69,23 @@ static RetainPtr<CTFontDescriptorRef> findFontDescriptor(CFStringRef referenceUR
 
 RetainPtr<CTFontRef> createCTFont(CFDictionaryRef attributes, float size, CTFontDescriptorOptions options, CFStringRef referenceURL, CFStringRef desiredPostScriptName)
 {
-    auto fontDescriptor = adoptCF(CTFontDescriptorCreateWithAttributes(attributes));
+    auto desiredReferenceURL = adoptCF(static_cast<CFURLRef>(CFURLCreateWithString(kCFAllocatorDefault, referenceURL, nullptr)));
```

Comment:
Hmm, why do we need that `static_cast`? Doesn't `CFURLCreateWithString` already return a `CFURLRef`?